src/FSharp.Data.GraphQL.Server/Execution.fs

```diff
@@ -539,24 +539,10 @@ let private executeQueryOrMutation (resultSet: (string * ExecutionInfo) []) (ctx
                     }
                 | [String p], t ->
                     asyncVal {
-                        let resolve value =
```

Comment:
This piece of code can be safely removed, since it was trying to handle the wrong path built by the planning function.